examples/dashboard/app/assets/css/main.css

```
@@ -0,0 +1,6 @@
+@import "tailwindcss" theme(static);
+@import "@nuxt/ui";
+
+@theme static {
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)